### DISCUSSION TITLE
how are you using igel and which updates/features would you rather see in the future

### DISCUSSION THREAD

**Discussion by nidhaloff** [nidhaloff/igel#71] [General]
Hi, I'm opening this discussion to have a closer discussion with people, who are using igel. Let's discuss together which advantages/disadvantages igel have from your point of view and more importantly, which things would you change and which features would you want to see implemented in the future.

**Comment by CustomCmd** [comment]
The arrival of an export option to other formats like ONNX would be a good idea for me. And if it is possible, the integration of a library like [ArrayFire](https://github.com/arrayfire/arrayfire) would be the icing on the cake.

**Comment by Andrewwheeler89** [comment]
I would like to upgrade thousands of IGEL's to the latest version supported at my organization
